hoss/dimension_utilities.py

```diff
@@ -45,7 +45,7 @@ def is_index_subset(message: Message) -> bool:
 
     """
     return any(
-        rgetattr(message, subset_parameter, None) is not None
+        rgetattr(message, subset_parameter, None) not in (None, [])
```

Comment:
I think we need a new unit test (or more likely a sub test) here to make sure the empty list is correctly identified in this check. Probably also a sub test with a non-empty list, too, just to be sure.

[Here's the test for this function](https://github.com/nasa/harmony-opendap-subsetter/blob/main/tests/unit/test_dimension_utilities.py#L623). I think the easiest thing is to just add two new subtests.

Reply:
I think there are several subtests with non empty list. But I added 2 subtests for the empty and non empty list cases

Reply:
Yeah, looking in more detail, I think you are right about the non-empty list. But thanks for adding a couple of unit tests.